package.json

```diff
@@ -22,20 +22,19 @@
     "@alaskaairux/icons": "^4.10.1",
     "chalk": "^4.1.2",
     "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1"
+    "lit-html": "^1.4.1",
```

Comment:
I am confused... I thought that this was updated to a Lit 2.0 element? The Lit dependency is being supported by the wc-range-datepicker dependency.

Reply:
Good catch, this is updated now.